### ISSUE TITLE
Validation loss and accuracy during training

### ISSUE THREAD

**Issue by tomoyaTT55** [zhangfaen/finetune-Qwen2-VL#34]
Hello. I would like to output not only the train loss but also the validation loss and accuracy in finetune.py. Could you provide me with an effective code for this?
